baseplate/clients/redis.py

```diff
@@ -57,6 +59,9 @@ def pool_from_config(
     if options.socket_timeout is not None:
         kwargs.setdefault("socket_timeout", options.socket_timeout.total_seconds())
 
+    if "queue_class" not in kwargs:
+        kwargs["queue_class"] = queue.LifoQueue
```

Comment:
It may be worth defaulting to `queue.Queue`. I'm adding this as an option for Thrift in https://github.com/reddit/baseplate.py/pull/645.

My analysis of traffic says that LIFO is very bad for load-balancing.

Reply:
Since this isn't the redis-cluster client, does load balancing come into play here? Usually there's only a single primary node that we want to talk to anyway so the downsides of maxing out connection pool sizes early on may not be worth it.

Reply:
Hmmm, good question. Even with non-cluster, there are still read-only replica options. So it may still be relevant to distribute the load for read requests.